### PR TITLE
Refine and clean up stylesheets

### PR DESCRIPTION
**Pull request by tysongach**
- Add percentage unit for color functions
- Change $header- variable name to $heading- (this better aligns with the HTML element naming—`H1` is heading 1, not header 1)
- Change keyword font weights to number values
- Remove fraction font feature settings from body
- Remove prefixes for box-sizing and transition
- Alphabetize properties

### CODE REVIEW

File: app/assets/stylesheets/_forms.scss [hunk 1]
```diff
@@ -26,23 +26,23 @@ label {
...
-textarea,
 #{$all-text-inputs},
-select[multiple=multiple] {
-  @include box-sizing(border-box);
-  @include transition(border-color);
-  background-color: white;
-  border-radius: $base-border-radius;
+select[multiple=multiple],
+textarea {
+  background-color: #fff;
```
Comment (kylefiedler) [inline app/assets/stylesheets/_forms.scss]: Isn't this why you and @Magnus-G had me put `$base-background-color` back in?

File: app/assets/stylesheets/_base.scss [hunk 2]
```diff
@@ -3,15 +3,13 @@
...
 // @import "grid-settings";
 
-// Typography and Elements
 @import "typography";
-@import "forms";
-@import "tables";
 @import "lists";
+@import "forms";
 @import "buttons";
+@import "tables";
```
Comment (kylefiedler) [inline app/assets/stylesheets/_base.scss]: Should we alphabetize these instead?

**Comment by kylefiedler** [comment]
Two comments otherwise :+1:

File: app/assets/stylesheets/_forms.scss (hunk 1 above)
Comment (Magnus-G) [inline app/assets/stylesheets/_forms.scss]: Yeah! I told you it's needed! :sunglasses:

**Pull request merged**